react/BaseContent.tsx

```diff
@@ -27,6 +27,11 @@ const Content: FC<Props> = ({ finishShoppingButtonLink }) => {
   const handles = useCssHandles(CSS_HANDLES)
   const { variation } = useMinicartState()
   const { url: checkoutUrl } = useCheckoutURL()
+  const { navigate } = useRuntime()
+
+  const goToCheckout = (url: string) => {
+    navigate({ to: url })
+  }
```

Comment:
Navigation seems to be broken if the new checkout is not installed. I believe you should not use `navigate` in this case.